tsl/src/continuous_aggs/repair.c

```diff
@@ -108,6 +108,11 @@ cagg_rebuild_view_definition(ContinuousAgg *agg, Hypertable *mat_ht, bool force_
 			 NameStr(agg->data.user_view_schema),
 			 NameStr(agg->data.user_view_name));
 
+	/*
+	 * Partials are not supported anymore, rebuild cagg with new format.
+	 */
+	finalized = true;
+
```

Comment:
You can't do it because the old and new format are incompatibles. Users should migrate the CAgg using `cagg_migrate` procedure in order to make it work properly.

Here in repair I should say we can raise an WARNING if the CAgg is not finalized and do nothing.

Reply:
Ohh my bad. Updated repair to emit a warning message in case of old format caggs.